api/capi/src/nnstreamer-capi-single.c

```diff
@@ -522,9 +522,10 @@ ml_single_open_custom (ml_single_h * single, ml_single_preset * info)
   ml_tensors_info_s *in_tensors_info, *out_tensors_info;
   ml_nnfw_type_e nnfw;
   ml_nnfw_hw_e hw;
-  const char *fw_name;
+  const gchar *fw_name;
   gchar **list_models;
   guint num_models;
+  gchar *hw_name;
```

Comment:
Make the type name consistent. (look at line 347 of nnstreamer-capi-private.h)

Reply:
done.